src/components/PasswordComplexity.tsx

```diff
@@ -11,13 +12,13 @@ const PasswordComplexityDiv = styled.div`
   gap: 4px;
 `;
 
-const PasswordRequirementDiv = styled.div<{ met: boolean }>`
+const PasswordRequirementDiv = styled(P)<{ met: boolean }>`
```

Comment:
was this supposed to be changed to P, or did you want to keep it as a div?

Reply:
could not tell u why i randomly decided to do this .... THANKS CATHERINE